package.json

```diff
@@ -10,7 +10,7 @@
     "build": "nest build",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
     "start": "nest start",
-    "start:dev": "nest start --watch",
+    "start:dev": "NODE_ENV=development nest start --watch",
```

Comment:
Yeah, but the .env file is not shared with all of us.
So, if someone will follow-up and continue using this, the default environment that will be used is the development one.

Reply:
@Sahl-A 
I figured out that we will need to explicitly add ```NODE_ENV=development``` in the command of  ```"start:dev"```
The following use-case occurred with me:
Sometimes I would explicitly need to know the name of the environment I'm currently at, for example in the ```generateErrorBasedOnCurrentEnvironment``` function.
Because depending on the current env I'm working at, I'll do specific actions., because If there's no env name sent for me, I will not be able to distinguish between whether nothing was sent for me or I was given a wrong env name?

P.S: I know we currently need only production and development environments, but I think in the future(or to be more generic) we may need others such as test environment.

Reply:
lgtm